### PR TITLE
AuditTrailModel merge from gitlab repo

### PR DESCRIPTION
**Pull request by RegoAttainia**
This is the merge into github the changes from the gitlab version of this project.  After this gets merged in I will remove the gitlab project.

@gsiegman @mgonzo @AlexMaskovyak @atomboulian @TimGraf @arizonatribe please review

### CODE REVIEW

**Comment by TimGraf** [comment]
LGTM

File: attainia_django_extensions/models.py [hunk 1]
```diff
@@ -1,12 +1,49 @@
...
 class AuditTrailModel(models.Model):
-    created_at = models.DateTimeField(auto_now_add=True)
-    modified_at = models.DateTimeField(auto_now=True)
+    """ used to add change timestamps and users
+
+    If you don't explicitly specify a created_at or modified_at value, those time stamps
+    will be auto assigned on save (no need to do you own book keeping)
+    """
+    created_at = models.DateTimeField(blank=True, null=True)
+    modified_at = models.DateTimeField(blank=True, null=True)
```
Comment (TimGraf) [inline attainia_django_extensions/models.py]: I know @gsiegman has some comments about this but I think for now he was good with it.

Comment (AlexMaskovyak) [reply attainia_django_extensions/models.py]: This isn't the standard django way to do this, are we doing this so that they can be assigned on creation?

Comment (RegoAttainia) [reply attainia_django_extensions/models.py]: Yes, we need to ability to specify created_at-s and modified_at-s that not of the time of object creation.

**Comment by AlexMaskovyak** [comment]
There's an alternative way to solve the internal business problem. Closing this merge request.

**Pull request closed without merging**